Documentation/ABI/JSON.md

```diff
@@ -210,6 +210,12 @@ sufficient information to display the event in a human-readable format.
   "fail" | "difference" | "warning" | "details"
 ```
 
+<!--
```

Comment:
Not critical for this PR, but let's think about starting to draft the change list for ABIv1.

Reply:
I had that thought as well; I've taken an initial stab at that in `JSON.md`.